server/controllers/BookController.js

```diff
@@ -75,6 +75,55 @@ class BookController {
       return responseMessage(response, 500, { message: error.message });
     }
   }
+
+  /**
+* Edit Books
+* @name editBook
+* @param {Object} request
+* @param {Object} response
+* @returns {JSON} object
+* @memberof BookController
+*/
+  static async editBook(request, response) {
+    const { id } = request.params;
+    const {
+      title, description, isbn, price, yearPublished, authorName
+    } = request.body;
+    try {
+      const books = await Book.findOne({ where: { id: +id } });
+      const getAuthor = await BookAuthor.findOne({ where: { bookId: +id }, attributes: ['authorId'] });
+      if (!books) return responseMessage(response, 400, { message: 'book does not exist' });
+      Book.update({
+        title, description, isbn, price, yearPublished
```

Comment:
I observed that there is no condition to check if the same ISBN already exist because ISBN is meant to be unique

Reply:
done